docs/1.2-Shadow-with-Docker.md

```diff
@@ -1,81 +1,25 @@
 This page describes how to set up Shadow using Docker; this installation method is optional.
 
-## Building your own docker image
+## Run from the official Dockerfile
 
-There are many ways to do this.  Below, we base our install off of Fedora 32.
+1. Install docker from https://docs.docker.com/engine/install/.
+2. Build container:
+```
+git clone https://github.com/shadow/shadow.git
+cd shadow
+docker build . -t shadow --shm-size="1g"
+```
+3. Run tests:
+```
+docker run --shm-size="1g" --privileged --rm --entrypoint /src/ci/container_scripts/test.sh shadow
+```
 
-1. Get the Fedora repository via `docker pull fedora` This may take a few minutes.
-1. Start a container via `docker run -i -t fedora:32 --shm-size=1g /bin/bash`. The `--shm-size` option sets the size of the `/dev/shm` mount point within the container.
-1. Create a non-root user:
+## Run a simulation
 
-	```bash
-	useradd -s /bin/bash -g wheel shadow
-	dnf install -y passwd sudo
-	passwd shadow  # when asked, set the password to something sensible
-	```
+To be able to run a simulation you have to mount a volume with the simulation dependencies (configurations and binaries). This will generate a log directory owned by `root`.
 
-1. Install Shadow's dependencies and recommended tools (this may take a few minutes):
+For example, the next command runs the `shadow.config.xml` simulation present in the current path:
+```
+docker run --shm-size="1g" --privileged --rm --log-driver=none -v $(pwd):/src/ --entrypoint "/bin/bash" shadow -c "shadow --interpose-method=ptrace -l debug shadow.config.xml"
+```
```

Comment:
With the above change to the `ENTRYPOINT`, this line can change to:

```bash
docker run --shm-size="1g" --privileged --rm --log-driver=none -v $(pwd):/src/ shadow --interpose-method=ptrace -l debug shadow.config.xml
```

Reply:
Yes I forgot to update thank you :)